### ISSUE TITLE
German umlaut keys (ü, ä ...) and Eszett key (ß)

### ISSUE THREAD

**Issue by Cash-coder** [sezanzeb/input-remapper#874]
When I try to remap umlaut german keys I get: 
german ä": The output_symbol "ä" is not a macro and not a valid keycode-name"

I've tried with key_ä and symbol:ä, not working

logs 

1. `input-remapper-control --version`
input-remapper 2.0.1  https://github.com/sezanzeb/input-remapper
python-evdev 1.6.1

3. which linux distro (ubuntu 20.04, manjaro, etc.)
Fedora release 39 (Thirty Nine)

4. `echo $XDG_SESSION_TYPE`
wayland

5. which desktop environment (gnome, plasma, xfce4, etc.)
gnome + hyprland as window manager

6. `sudo ls -l /proc/1/exe`
lrwxrwxrwx. 1 root root 0 Apr 17 21:06 /proc/1/exe -> /usr/lib/systemd/systemd

7. paste the affected preset .json file from ~/.config/input-remapper-2/presets

8. `sudo pkill -f input-remapper-service && input-remapper-gtk -d`, apply the preset and hit your key. Then share that log.
21:26:22.208794 30240 GUI DEBUG message_broker.py:76: from controller.py:720: Signal=status_msg: StatusData(ctx_id=5, msg='"Select Word": Macro.add_hold_keys() got an unexpected keyword argument \'LeftShift_L\'', tooltip="Macro.add_hold_keys() got an unexpected keyword argument 'LeftShift_L'")
21:26:22.208978 30240 GUI DEBUG message_broker.py:76: from controller.py:720: Signal=status_msg: StatusData(ctx_id=5, msg='"Select Line": Unknown key "EndShift_L"', tooltip='Unknown key "EndShift_L"')
21:26:22.209307 30240 GUI DEBUG message_broker.py:76: from controller.py:720: Signal=status_msg: StatusData(ctx_id=5, msg='"german ä": The output_symbol "ä" is not a macro and not a valid keycode-name', tooltip='The output_symbol "ä" is not a macro and not a valid keycode-name')
21:26:22.209609 30240 GUI DEBUG message_broker.py:76: from data_manager.py:136: Signal=selected_event: InputConfig KEY_A

**Comment by maddinpsy** [comment]
I managed to get this setting the compose key in system settings to ALT_R.
German Ä can be made with ALT_R + Shift_R + KEY_APOSTROPHE + A
See this post for details: https://github.com/xremap/xremap/issues/120#issuecomment-1518699015

**Comment by oleasteo** [comment]
Using a compose key is no option for me, as I desire the comfort of the pure US-layout ALT_R key (e.g. for one-handed backward/forward history navigation in browsers).

Adding special character input key combinations seems like a standard use-case for a custom keyboard mapping o.O

**Comment by jonasBoss** [comment]
I am not sure if I understand the issue correctly. You are using a US-Keyboard layout in your desktop environment, but want to inject the German "ä" key?

I think that is only possible by injecting a combination (See [Limitations](https://github.com/sezanzeb/input-remapper/blob/main/readme/usage.md#limitations): _[...] Furthermore, if your configured keyboard layout doesn't support the special character at all (not even via a combination), then it also won't be possible for input-remapper to map that character at all.[...]_).

**Comment by oleasteo** [comment]
@jonasBoss Thanks for the clarification. You did understand my use-case correctly. Not sure if the OP had the same scenario; if not, maybe their issue would have been solved by @maddinpsy comment.

In my opinion, the first sentence of the limitations should definitely be included in the main readme. It appears to be crucial and pretty concise to understand this tool's scope.

For reference for anyone with my use-case: I found `keyd` to work for me, see [gist with my setup](https://gist.github.com/oleasteo/421802950cd767cfe87ec12b84e44150#file-readme-md)
